tf2/include/tf2/buffer_core_interface.h

```diff
@@ -50,6 +50,10 @@ namespace tf2
 class BufferCoreInterface
 {
 public:
+  TF2_PUBLIC
+  virtual
+  ~BufferCoreInterface() {}
```

Comment:
Is it more idiomatic to use `= default;` in this case? I honestly don't know, I'm just curious.

Reply:
`= default` is definitely more idiomatic, I forgot about using it.

The only difference I know is that a class with a defaulted destructor is trivially destructible, and one with an empty destructor is not.
In this case it doesn't apply, as classes with virtual destructors are never trivially destructible, i.e. I think that a empty virtual destructor is exactly equivalent to the defaulted one. CPP is too complex so there might be another difference :joy:.

Reply:
fixed in https://github.com/ros2/geometry2/pull/272/commits/143226d1fafc6e3a950cff010f2846849387fc92.